spec/site/connect_spec.rb

```diff
@@ -51,22 +51,52 @@ def check_sequence version
     end
   end
 
+# Verify that the handshake sequence upon connection is correct for 3.1.1
+#
+# 1. Given the site is new, connected and version 3.1.1
+# 2. Send and receive handshake messages
+# 3. Expect the handshake messages to be in the specified sequence corresponding to version 3.1.1
+# 4. Expect the site to be ready
```

Comment:
the 'site' object is a local proxy for the real equipment.
what we expect is that the site proxy object reaches the ready state. this means that the connection sequence is complete. i suggest "expect the connection sequence to be complete"

Reply:
Right, that makes sense. I'll change it.